.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 The original authors.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
+name: Build
+
+env:
+  MAVEN_ARGS: -B -e
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    name: Java ${{ matrix.java }} / Maven (${{ matrix.profile }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8,11,17]
```

Comment:
I think this is too much. Maybe, Java 11 and 17 only? Wdyt?

Reply:
Given, that we will keep 1.8 around (see my commment above I think that we should also keep it here).